src/core/server/models/tenant/helpers.ts

```diff
@@ -24,9 +24,9 @@ export const getDefaultReactionConfiguration = (
   icon: "thumb_up",
 });
 
-export const getDefaultStaffConfiguration = (
+export const getDefaultBadgeConfiguration = (
   bundle: FluentBundle
-): Tenant["staff"] => ({
+): Tenant["badges"] => ({
```

Comment:
Should a default badge configuration be added here as well for `memberLabel` set to `Member`?